eclcli/dns/v2/record.py

```diff
@@ -153,9 +153,14 @@ def get_parser(self, prog_name):
             help="ID of the zone which recordset you want to show belong to.",
         )
         parser.add_argument(
-            "recordset_id",
-            metavar="<record_id>",
-            help="ID of the recordset you want to show.",
+            "recordset_id_or_name",
+            metavar="<record_id_or_name>",
+            help="ID or name of the recordset you want to show.",
+        )
+        parser.add_argument(
```

Comment:
As I mentioned above, show command must return single record. Pagination does not make sense.